### PR TITLE
Added a dev tiltfile to repo

### PR DESCRIPTION
**Pull request by ICoxCH**

### CODE REVIEW

File: Tiltfile.dev [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  command = 'mvn compile jib:dockerBuild -Dimage=$EXPECTED_REF',
+  live_update = [
+    sync(
+      local_path = './target/classes',
+      remote_path = '/app/classes'
+    ),
+    restart_container()
+  ],
+  deps = ['./target/classes']
+)
```
Comment (mattch1) [inline Tiltfile.dev:18]: Newline.

Comment (ICoxCH) [reply Tiltfile.dev:18]: Happy to add a new line, but what does the newline provide? Why is it needed

Comment (mattch1) [reply Tiltfile.dev]: I think sonar complains about it; github always does.

**Review by mattch1 — APPROVED** [review]

File: Tiltfile.dev (hunk 1 above)
Comment (ICoxCH) [reply Tiltfile.dev]: Sonar didn't raise it as a smell, bug, hotspot or vulnerability so I think Sonar is ok with it.
And the Red circle on Github is less complaining and more letting you know. I am yet to ever encounter an issue caused by this

Comment (mattch1) [reply Tiltfile.dev]: In that case it's just a styling preference rather than anything else.

**Pull request merged**